src/miner.cpp

```diff
@@ -826,7 +815,6 @@ void StakeMiner(CWallet *pwallet)
         MiningCPID BoincData;
         { LOCK(MinerStatus.lock);
             //clear miner messages
-            MinerStatus.Message="";
```

Comment:
error here. not clearing message will result in runaway allocation. either clear it or remove Message from struct completely.